.github/workflows/image-arm.yaml

```diff
@@ -38,6 +38,8 @@ jobs:
           platforms: all
       - name: Install Cosign
         uses: sigstore/cosign-installer@main
+      - name: Login to Quay Registry
```

Comment:
we _don't_ have to push anything in the registry while building PRs

Reply:
added condition to push only on `main`.